src/Publisher/AbstractCloudPublisher.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Lmc\Steward\Publisher;
+
+use Facebook\WebDriver\Remote\RemoteWebDriver;
+use Lmc\Steward\Test\AbstractTestCaseBase;
+
+/**
+ * Abstract publisher for cloud services with HTTP API to store test results
+ */
+abstract class AbstractCloudPublisher extends AbstractPublisher
+{
+    /** @var string Content type of the request */
+    const CONTENT_TYPE = 'application/x-www-form-urlencoded';
+
+    public function publishResults(
+        $testCaseName,
+        $status,
+        $result = null,
+        \DateTimeInterface $startDate = null,
+        \DateTimeInterface $endDate = null
+    ) {
+        // we publish only result for each separate test using publishResult()
```

Comment:
IMHO it would be better to throw an Exception.

Because when user call this funciton nothing will happen. And now what? Did I misconfigure something?

Or I don`t get it at all :-)